setup.py

```diff
@@ -21,19 +21,20 @@
 DESC = 'Kriging Toolkit for Python'
 LDESC = 'PyKrige is a kriging toolkit for Python that supports two- and ' \
         'three-dimensional ordinary and universal kriging.'
-PACKAGES = ['pykrige']
+PACKAGES = ['pykrige', 'pykrige.optimise']

```

Comment:
Why would you put such a restriction? It just seems natural to add something like this in a subpackage, as not too many people will use this.